testHelper/test.utilities.ts

```diff
@@ -0,0 +1,3 @@
+import { queryByAttribute } from "@testing-library/react";
+
+export const getById = queryByAttribute.bind(null, "id");
```

Comment:
This is kind of a hack maybe but I don't see how you operate in this library without this. You can use test id as well. 
https://stackoverflow.com/a/53003981/3058839

Reply:
Could this become easier with React 18 updates? 🤔 

https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis

Reply:
I think it's more of a testing library limitation.